kraken-app/kraken-app-hub/src/test/resources/mock/mapping-matrix/mapping.matrix.quote.eline.add.yaml

```diff
@@ -13,6 +13,7 @@ spec:
       - name: enabled
         expectType: EXPECTED
         value: false
+        errorMsg: "instantSyncQuote==false"
```

Comment:
errorMsg is a bit confused, should be a more reasonable message.

Reply:
Thanks. The tips have been redefined.